models/library.go

```diff
@@ -53,19 +63,56 @@ type Library struct {
 	// The Path to the library in the container image. Empty string when Lockfile scan.
 	// This field is used to convert the result JSON of a `trivy image` using trivy-to-vuls.
 	FilePath string
+	Digest   string
+}
+
+func trivyInit() error {
+	if err := tlog.InitLogger(config.Conf.Debug, config.Conf.Quiet); err != nil {
+		return xerrors.Errorf("Failed to init trivy logger. err: %w", err)
+	}
+
+	javadb.Init(config.Conf.TrivyCacheDBDir, config.Conf.TrivyJavaDBRepository, config.Conf.TrivySkipJavaDBUpdate, config.Conf.Quiet, ftypes.RegistryOptions{})
+	return nil
+}
+
+var trivyInitOnce = sync.OnceValue(trivyInit)
+
+func trivyInitJavaDB() error {
+	javadb.Init(config.Conf.TrivyCacheDBDir, config.Conf.TrivyJavaDBRepository, config.Conf.TrivySkipJavaDBUpdate, config.Conf.Quiet, ftypes.RegistryOptions{})
+
+	javaDBClient, err := javadb.NewClient()
```

Comment:
How about implementing the javaDB download as a continuation of the trivy.db download part?
https://github.com/future-architect/vuls/blob/74aa70a24ac7723f68d37c6f39fcb13285a1e3b9/detector/library.go#L34

Reply:
Also, the created client should be closed appropriately.
https://pkg.go.dev/github.com/aquasecurity/trivy@v0.49.1/pkg/javadb#DB.Close